test/extensions/filters/network/local_ratelimit/local_ratelimit_test.cc

```diff
@@ -0,0 +1,199 @@
+#include "envoy/config/filter/network/local_rate_limit/v2alpha/local_rate_limit.pb.validate.h"
+
+#include "common/stats/isolated_store_impl.h"
+
+#include "extensions/filters/network/local_ratelimit/local_ratelimit.h"
+
+#include "test/mocks/event/mocks.h"
+#include "test/mocks/network/mocks.h"
+#include "test/mocks/runtime/mocks.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
+using testing::InSequence;
+using testing::NiceMock;
+using testing::Return;
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace LocalRateLimitFilter {
+namespace {
+
+class LocalRateLimitTestBase : public testing::Test {
+public:
+  void setup(const std::string& filter_yaml) {
```

Comment:
setup -> initialize?
I think we have a general pattern of SetUp being called automatically and initialize() being manual